docs/arch/adr-xx.md

```diff
@@ -0,0 +1,65 @@
+
+- RHC: A right handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the +y axis with angle=0 along the +x axis
+
+- LHC: A left handed coordinate system, this is defined as a system where positive angles rotate an object from the +x axis to the -y axis with angle=0 along the +x axis
+
+- Domain: The range of values a number can take [ or ] is inclusive, ( or ) is exclusive.
+
+- Upper origin: The point 0, 0 is in the upper left of the plot
+
+- Lower origin: The point 0, 0 is in the lower left of the plot
+
+The inconsistencies comprise of:
+
+- The browser SVG coordinate systems use _RHC_ with an _upper origin_ resulting in positive angles rotating clockwise. Most plotting software (R, Python, Matlab) are _RHC_ with a _lower origin_ resulting in positive angles rotating counter-clockwise. 
+  - The position of origin has been inconsistent between tools which has an effect on the final annotation too. Most use the center x, y point, but some don't
+  - Some of the drawing tools use _LHC_
+- Some tools' annotation use `angle` some use `rotation`
+- It's unclear when the x, y annotation refers to the center point of the shape
+- It's unclear when the x, y annotation is being used as the point of rotation
+
+Some of the mark annotation models have a few other issues as well:
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` or `[0, 360)` for consistent angle calculation
+- The annotation model should use `angle` for naming
+- The annotation model should consistency use `x_center` and `y_center` for shapes where that is applicable
+  - The exceptions are point, line, fan. The fan tool is not symmetric. 
+  - Shape clustering in aggregation is always done with the center
+- All rotations should be defined about `x_center` and `y_center` point. If the rotation cannot be defined around the center point, then the point used that should be clearly recorded in the annotation as `x_rotation` and `y_rotation`
```

Comment:
From a developer's point of view, I'd prefer not to have logic in the code along the lines of `if x elseif x_center elseif x_rotation`. Here, for example: https://github.com/zooniverse/front-end-monorepo/blob/df5b630209dc65e658865b2f75a2657df490241e/packages/lib-classifier/src/plugins/drawingTools/components/DrawingToolRoot/DrawingToolRoot.js#L59-L61

Reply:
`x_center` or `x_rotation` will be replacing the non-specific `x` and same for `y`. I think this is easily avoided using the a computed view function on the model. I'll update the description here to include this suggestion.